jsonschema-core.xml

```diff
@@ -1163,37 +1167,19 @@
                     </t>
                     <t>
-                        The "$schema" keyword SHOULD be used in a resource root schema.
-                        It MUST NOT appear in resource subschemas.  If absent from the root schema, the
-                        resulting behavior is implementation-defined.
+                        The "$schema" keyword SHOULD be used in the document root schema object,
+                        and MAY be used in the root schema objects of embedded schema resources.
+                        It MUST NOT appear in subschemas.  If absent from the document root schema,
+                        the resulting behavior is implementation-defined.
                     </t>
                     <t>
-                        If multiple schema resources are present in a single document, then all
-                        schema resources SHOULD Have the same value for "$schema".  The result of
-                        differing values for "$schema" within the same schema document is
-                        implementation-defined.
-                        <cref>
-                            Using multiple "$schema" keywords in the same document would imply that the
-                            feature set and therefore behavior can change within a document.  This would
-                            necessitate resolving a number of implementation concerns that have not yet
-                            been clearly defined.  So, while the pattern of using "$schema" only in root
-                            schemas is likely to remain the best practice for schema authoring,
-                            implementation behavior is subject to be revised or liberalized in
-                            future drafts.
-                        </cref>
-                        <cref>
-                            The exception made for embedded schema resources is to
-                            allow bundling multiple schema resources into a single schema document
-                            without needing to change their contents, as described later in this
-                            specification.
-                        </cref>
-                        <!--
-                            In particular, the process of validating an instance, including validating a
-                            schema as an instance against its meta-schema, only allows for a single set
-                            of rules across the entire instance document.  There is no equivalent of
-                            changing the meta-schema partway through the validation for non-schema
-                            instances.
-                        -->
+                        If multiple schema resources are present in a single document, then
+                        schema resources which do not have a "$schema" keyword in their root
+                        schema object inherit the meta-schema from the enclosing resource.
```

Comment:
do we need a "MUST" (or "SHOULD"?) here?

Reply:
MAYBE

Reply:
TBH I'm never entirely sure, this feels like a thing that just is.  But "MUST be treated as if they declared the same `$schema` as the enclosing resource" or something like that probably wouldn't hurt.

Reply:
Yes, pleas make this change. Then a requirement keyword is picked up and can have a test! =]